src/cascadia/TerminalControl/TermControl.h

```diff
@@ -208,6 +212,7 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         std::unique_ptr<::Microsoft::Console::Render::UiaEngine> _uiaEngine;
 
         IControlSettings _settings;
+        IControlSettings _unfocusedAppearance;

```

Comment:
```suggestion
        IControlAppearance _unfocusedAppearance;
```
Shouldn't this be an `IControlAppearance`?